### PR TITLE
Add image block's default content

### PR DESCRIPTION
**Pull request by GeronimoJohn**
- Add the default content for add image block
   - Content placeholder: grey block with image icon

### CODE REVIEW

File: libs/journeys/ui/src/components/Image/Image.tsx [hunk 1]
```diff
@@ -29,13 +29,23 @@ export function Image({
...
+        <NextImage
+          src={src}
+          alt={alt}
+          height={height}
+          width={width}
+          objectFit="cover"
+        />
+      ) : (
+        <NextImage
+          src={`${window.location.origin}/DefaultImageIcon.png`}
```
Comment (tataihono) [inline libs/journeys/ui/src/components/Image/Image.tsx:42]:
```suggestion
          src="/DefaultImageIcon.png"
```

File: libs/journeys/ui/src/components/Image/Image.spec.tsx [hunk 2]
```diff
@@ -23,4 +23,13 @@ describe('Image', () => {
       'random image from unsplash'
     )
   })
+
+  it('should render the default image', () => {
+    const { getByRole } = render(<Image {...block} src={null} alt={'defaultImageIcon'} />)
```
Comment (tataihono) [inline libs/journeys/ui/src/components/Image/Image.spec.tsx:28]: Please use the notation below.
```suggestion
    const { getByRole } = render(<Image {...block} src={null} alt="defaultImageIcon" />)
```

File: apps/journeys-admin/src/components/Editor/ControlPanel/BlocksTab/Image/Image.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { MockedProvider } from '@apollo/client/testing'
+import { render } from '@testing-library/react'
+import { Image } from '.'
+
+describe('Image', () => {
```
Comment (tataihono) [inline apps/journeys-admin/src/components/Editor/ControlPanel/BlocksTab/Image/Image.spec.tsx:5]: need to add in a spec to test the mutation is being called correctly.

**Review by tataihono — CHANGES_REQUESTED** [review]

File: apps/journeys-admin/src/components/Editor/ControlPanel/BlocksTab/Image/Image.tsx [hunk 4]
```diff
@@ -0,0 +1,87 @@
+import { gql, useMutation } from '@apollo/client'
+import {
+  ActiveTab,
+  EditorContext,
+  IMAGE_FIELDS,
+  transformer,
+  TreeBlock
+} from '@core/journeys/ui'
+import InsertPhotoRounded from '@mui/icons-material/InsertPhotoRounded'
+import { useJourney } from 'apps/journeys-admin/src/libs/context'
```
Comment (tataihono) [inline apps/journeys-admin/src/components/Editor/ControlPanel/BlocksTab/Image/Image.tsx:10]: can't import like this

**Review by tataihono — CHANGES_REQUESTED** [review]

File: apps/journeys-admin/src/components/Editor/ControlPanel/BlocksTab/BlocksTab.tsx [hunk 5]
```diff
@@ -22,7 +22,7 @@ export function BlocksTab(): ReactElement {
         }}
       >
         <Typography />
-        <Button icon={<InsertPhotoRounded />} value="Image" />
+        <Image />
```
Comment (tataihono) [inline apps/journeys-admin/src/components/Editor/ControlPanel/BlocksTab/BlocksTab.tsx:25]: I think the component needs to be called NewImageButton

**Review by tataihono — DISMISSED** [review]

File: apps/journeys-admin/src/components/Editor/ControlPanel/BlocksTab/BlocksTab.tsx (hunk 5 above)
Comment (GeronimoJohn) [reply apps/journeys-admin/src/components/Editor/ControlPanel/BlocksTab/BlocksTab.tsx]: Should this naming pattern also be applied to the other add button components? e.g. NewTypographyButton, NewPollButton

**Review by tataihono — APPROVED** [review]

**Pull request merged**